politeiawww/cmd/shared/client.go

```diff
@@ -1135,14 +1135,14 @@ func (c *Client) GetAllVetted(gav *www.GetAllVetted) (*www.GetAllVettedReply, er
 }
 
 // NewComment submits a new proposal comment for the logged in user.
-func (c *Client) NewComment(nc *www.NewComment) (*www.NewCommentReply, error) {
+func (c *Client) NewComment(nc *pi.CommentNew) (*pi.CommentNewReply, error) {
```

Comment:
Since we're updating this, lets not pass in a pointer anymore.  There's not really any need to.  My rule for passing in structs vs points is, if you're updating any of the the fields then pass in a pointer, otherwise pass in a struct.

Also, lets update the name of the function to be consistent with the name of the route.

Reply:
Will do, makes sense, thanks for clarifying the pointer vs. struct.